tailwind.config.js

```diff
@@ -1,19 +1,23 @@
 module.exports = {
-  purge: {
-    mode: 'all',
-    content: ['./**/*.html'],
-    options: {
-      whitelist: [],
-    },
-  },
-  theme: {
-    container: {
-      center: true,
-    },
-    extend: {
-      colors: {},
-    },
-  },
-  variants: {},
-  plugins: [require('@tailwindcss/typography')],
+	purge: {
+		mode: 'all',
+		content: ['./**/*.html'],
+		options: {
+			whitelist: [],
+		},
+	},
+	theme: {
+		container: {
+			center: true,
+		},
+		extend: {
+			colors: {},
+		},
+	},
+	variants: {
+		extend: {
+			zIndex: ['hover'],
```

Comment:
This should fix [Codacy Linting](https://app.codacy.com/gh/web3community/web3community.github.io/pullRequest?prid=8517714) warnings. 🙂
```suggestion
			zIndex: ["hover],
```